packages/internal/bridge/sdk/src/tokenBridge.ts

```diff
@@ -384,39 +384,40 @@ export class TokenBridge {
     const currentBridgeMethods = await this.getBridgeMethods(direction);
     const bridgeContract = await this.getBridgeContract(direction);
 
+    let functionName: string;
+    let parameters: any[];
     /**
      * Handle bridge transaction for native token
      */
     if (token.toUpperCase() === NATIVE) {
       if (sender === recipient) {
```

Comment:
should we handle scenario where a checksummed version of an address might evaluate different from a non-checksummed version?

Reply:
Yep. Ticket created